flow-typed/npm/gray_matter_vx.x.x.js

```diff
@@ -0,0 +1,8 @@
+
+// @flow
+// gray-matter flow-typed version based on https://github.com/jonschlinkert/gray-matter
+
+declare module 'gray-matter' {
+  declare module.exports: any;
```

Comment:
Not super proud of these flow typings, but they're all backed by Typescript definitions as well.

Reply:
Yikes. Can we get better type definitions for these?

Reply:
Added